src/app/widgets/JogControl/index.jsx

```diff
@@ -256,7 +256,7 @@ class AxesWidget extends PureComponent {
         jog: (params = {}) => {
             const { units } = this.state;
             const modal = (units === METRIC_UNITS) ? 'G21' : 'G20';
-            const s = map(params, (value, letter) => ('' + letter.toUpperCase() + value)).join(' ');
+            const s = map(params, (value, letter) => ('' + (letter === 'a' || letter === 'A' ? 'Y' : letter.toUpperCase()) + value)).join(' ');
```

Comment:
Don't think you need to do a check here, you should probably just change the letter property value from params to whatever axis is needed before, when you call the jog function.